hosted_control_planes/hcp-machine-config.adoc

```diff
@@ -10,6 +10,11 @@ In a standalone {product-title} cluster, a machine config pool manages a set of
 
 In {hcp}, the `MachineConfigPool` CR does not exist. A node pool contains a set of compute nodes. You can handle a machine configuration by using node pools.
 
+[NOTE]
+====
+In {product-title} 4.18 and later, the default container runtime for worker nodes is changed from runC to crun.
```

Comment:
```suggestion
In {product-title} 4.18 or later, the default container runtime for worker nodes is changed from runC to crun.
```

According to IBM Style: "To state a requirement that includes earlier or later versions of a product, use the phrases _or earlier_ to indicate earlier versions and _or later_ to indicate subsequent versions."

Reply:
From a search of the published docs, I see that `runC` and `crun` are normally written without backticks, for example: https://docs.openshift.com/container-platform/4.16/nodes/containers/nodes-containers-using.html